frontend/src/App.js

```diff
@@ -275,22 +275,22 @@ const CourseText = (
   <>
     <h1 dangerouslySetInnerHTML={{__html: page.title}}/>
     {page.steps.slice(0, step_index + 1).map((part, index) =>
-      <div key={index} id={`step-text-${index}`}>
+      <div key={index} id={`step-text-${index}`} className={index > 0 ? 'pt-3' : ''}>
         <Markdown html={part.text} copyFunc={text => bookSetState("editorContent", text)}/>
-        <hr/>
+        <hr style={{ margin: '0' }}/>
       </div>
     )}
     <Messages {...{messages}}/>
 
-    <div>
+    <div className='pt-3'>
```

Comment:
Please add a comment explaining what pt-3 means.

Reply:
pt-3 is a shorthand that is used in most css libraries and is found somewhere in here, though I could not determine where. It is shorthand for padding-top 3 which should equal to 1 rem (16px by deafult).

Reply:
I'm guessing it's coming from bootstrap, please just add this description in a comment in code.

Reply:
Yes, you are correct. Didn't realize bootstrap was used via link in index.html, I thought it was downloaded. The documentation is here, though I will add it in the coment: https://getbootstrap.com/docs/4.0/utilities/spacing/